site/docs/latest/cookbooks/tiered-storage.md

```diff
@@ -86,6 +93,67 @@ If you are running in EC2 you can also use instance profile credentials, provide
 
 {% include admonition.html type="warning" content="The broker must be rebooted for credentials specified in pulsar_env to take effect." %}
 
+#### Configuring the size of block read/write
+
+Pulsar also provides some knobs to configure the size of requests sent to AWS S3.
+
+- ```s3ManagedLedgerOffloadMaxBlockSizeInBytes```  configures the maximum size of a "part" sent during a multipart upload. This cannot be smaller than 5MB. Default is 64MB.
+- ```s3ManagedLedgerOffloadReadBufferSizeInBytes``` configures the block size for each individual read when reading back data from AWS S3. Default is 1MB.
+
+In both cases, these should not be touched unless you know what you are doing.
+
+
+### Configuring for "google-cloud-storage" driver
+
+#### Configuring the Bucket
+
+On the broker, the administrator must configure the bucket and credentials for the cloud storage service. The configured bucket and credentials must exist before attempting to offload. If it does not exist, the offload operation will fail.
+
+Regarding driver type "google-cloud-storage", the administrator should configure `gcsManagedLedgerOffloadBucket`.
+
+```conf
+gcsManagedLedgerOffloadBucket=pulsar-topic-offload
+```
+
+#### Configuring the Bucket Region
+
+Bucket Region is the region where bucket located. 
```

Comment:
Is this a required config? What happens if it isn't configured. Is it required if the bucket has been explicitly created in a region? What are the tradeoffs between `us multi-regional location` and a single bucket?